core/js/typespecs/openai/messages.ts

```diff
@@ -77,9 +77,9 @@ export const chatCompletionMessageToolCallSchema = z.object({
 const chatCompletionAssistantMessageParamSchema = z.object({
   role: z.literal("assistant"),
   content: z.string().nullish(),
-  function_call: functionCallSchema.optional(),
+  function_call: functionCallSchema.nullish(),
```

Comment:
should we make the `name` nullish too?

Reply:
Yeah

Reply:
For future reference - we can't do this willy nilly since it breaks the openai wrappers

Reply:
What's the breakage?